src/components/Generic/Dialogs/AddOrUpdatePrinterDialog.vue

```diff
@@ -82,67 +79,45 @@
                         required
                       ></v-text-field>
                     </validation-provider>
-                  </v-col>
-                </v-row>
-
-                <v-expansion-panels accordion>
-                  <v-expansion-panel>
-                    <v-expansion-panel-header>Advanced settings</v-expansion-panel-header>
-                    <v-expansion-panel-content>
-                      <v-col cols="12" md="12">
-                        <validation-provider v-slot="{ errors }" name="PrinterHostPrefix">
-                          <v-select
-                            v-model="formData.printerHostPrefix"
-                            :error-messages="errors"
-                            :items="['http', 'https']"
-                            label="Insecure/Secure HTTP"
-                            required
-                            value="http"
-                          ></v-select>
-                        </validation-provider>
-                      </v-col>
-                    </v-expansion-panel-content>
-                  </v-expansion-panel>
-                </v-expansion-panels>
+                  </v-row>
+                </v-col>
               </v-container>
             </v-col>
 
             <PrinterChecksPanel v-if="showChecksPanel" :cols="4">
               <v-btn @click="showChecksPanel = false">Hide checks</v-btn>
             </PrinterChecksPanel>
           </v-row>
-          <v-row>
-            <v-col v-if="!isClipboardApiAvailable" cols="12">
-              Clipboard is not available. Copy or paste the following:
-              <br />
-              <v-textarea v-model="copyPasteConnectionString" rows="3"></v-textarea>
-            </v-col>
-          </v-row>
         </v-card-text>
         <v-card-actions>
-          <em class="red--text">* indicates required field</em>
-          <v-spacer></v-spacer>
-          <v-btn text @click="closeDialog()">Close</v-btn>
-          <v-btn :disabled="invalid" color="gray" text @click="duplicatePrinter()">
-            Duplicate
-          </v-btn>
-          <v-btn :disabled="invalid" color="warning" text @click="testPrinter()">
-            Test connection
-          </v-btn>
+          <v-container>
```

Comment:
I dont agree with the changes in the block below, could you let me know your thoughts? Is this maybe work-in-progress?

- Please revert this concerning the container and added divs.
- The buttons are ok
- v-row should not be added (in Vuetify2 v-row should not be left without v-col, but in this case I dont see a reason for it to be used here)
- empty v-card-actions should be removed

Reply:
I will review this block again. But my goal was to make it more responsive. Because before it would overflow on small screen.